### PR TITLE
feat: use latest vuepress and remove unused features

### PR DESCRIPTION
**Pull request by Mister-Hope**
see [comments](https://github.com/abapGit/docs.abapgit.org/pull/254#issuecomment-2545503106)

### CODE REVIEW

**Comment by larshp** [comment]
redirects are removed

File: src/.vuepress/theme.ts [hunk 1]
```diff
@@ -16,81 +15,128 @@ export default hopeTheme({
...
+        "/development/guidelines.html": "/development-guide/read-first/guidelines.html",
+        "/development/index.html": "/development-guide/",
+        "/development/serializers.html": "/development-guide/serializers/serializers.html",
+        "/development/technical-links.html":
+          "/development-guide/technical-details/technical-links.html",
+        "/development/tests.html": "/development-guide/testing/tests.html",
+        "/development/versioning.html": "/development-guide/read-first/versioning.html",
+        "/user-guide/projects/online/uninstall.html": "/user-guide/projects/remove/uninstall.html"
+      }
+    },
```
Comment (Mister-Hope) [inline src/.vuepress/theme.ts:126]: Redirects are moved here

**Comment by mbtools** [comment]
Hi @Mister-Hope 

Marc here. I had the same initial reaction to your PR as Lars. It came out of nowhere and has no description at all. This is a very large diff too and I didn't see the redirects either. You have to scroll way down to find them. 

Like Lars, I would have rejected this. 

Your account name looks rather anonymous, too. So it's not apparent that you are from the Vue team. Now that I saw your profile the PR makes more sense. Some short description here would have definitely avoided this misunderstanding. 

I will take a look at this update later in more detail.

**Comment by Mister-Hope** [comment]
The color is meant to fix the background/text contrast, so we made a update for this, technically now a set of theme colors are automatically generated from the given theme color.

Users can still override them in palette file, I will upgrade that now.

**Comment by Mister-Hope** [comment]
Ref: https://dequeuniversity.com/rules/axe/4.9/color-contrast

**Comment by mbtools** [comment]
That makes a lot of sense. I'm all for improving accessibility. Thanks again!

**Review by mbtools — APPROVED** [review]
👍

**Pull request merged**